### ISSUE TITLE
error

### ISSUE THREAD

**Issue by Cosplayeridol** [lewangdev/autotranslate#11]
RuntimeError                              Traceback (most recent call last)
[<ipython-input-5-163a91088c5e>](https://localhost:8080/#) in <cell line: 45>()
     43 fragments = []
     44 
---> 45 for segment in segments:
     46   print(f"[{seconds_to_time_format(segment.start)} --> {seconds_to_time_format(segment.end)}] {segment.text}")
     47   if word_level_timestamps:

1 frames
[/usr/local/lib/python3.10/dist-packages/faster_whisper/transcribe.py](https://localhost:8080/#) in encode(self, features)
    627         features = get_ctranslate2_storage(features)
    628 
--> 629         return self.model.encode(features, to_cpu=to_cpu)
    630 
    631     def generate_with_fallback(

RuntimeError: Library libcublas.so.11 is not found or cannot be loaded

**Comment by lewangdev** [comment]
### GPU

GPU execution requires the following NVIDIA libraries to be installed:

* [cuBLAS for CUDA 11](https://developer.nvidia.com/cublas)
* [cuDNN 8 for CUDA 11](https://developer.nvidia.com/cudnn)

There are multiple ways to install these libraries. The recommended way is described in the official NVIDIA documentation, but we also suggest other installation methods below.

<details>
<summary>Other installation methods (click to expand)</summary>

#### Use Docker

The libraries are installed in this official NVIDIA Docker image: `nvidia/cuda:11.8.0-cudnn8-runtime-ubuntu22.04`.

#### Install with `pip` (Linux only)

On Linux these libraries can be installed with `pip`. Note that `LD_LIBRARY_PATH` must be set before launching Python.

```bash
pip install nvidia-cublas-cu11 nvidia-cudnn-cu11

export LD_LIBRARY_PATH=`python3 -c 'import os; import nvidia.cublas.lib; import nvidia.cudnn.lib; print(os.path.dirname(nvidia.cublas.lib.__file__) + ":" + os.path.dirname(nvidia.cudnn.lib.__file__))'`
```

#### Download the libraries from Purfview's repository (Windows & Linux)

Purfview's [whisper-standalone-win](https://github.com/Purfview/whisper-standalone-win) provides the required NVIDIA libraries for Windows & Linux in a [single archive](https://github.com/Purfview/whisper-standalone-win/releases/tag/libs). Decompress the archive and place the libraries in a directory included in the `PATH`.

</details>

**Comment by Cosplayeridol** [comment]
Thank you for your reply, but I don’t know how to operate it. When I used it today, I found that the program became like this. Maybe you need to update the project code to adapt to colab?

**Comment by Cosplayeridol** [comment]
Your reply to me is that I need to download this resource package and put it in the colab folder by myself? Is there an easier way that can be solved by just entering the program code and running it? This convenience is one of the reasons why I use colab as a tool.....

**Comment by Cosplayeridol** [comment]
In addition, is it possible to support the translation function of gpt4?

**Comment by lewangdev** [comment]
Bad news, colab has upgraded their environment's CUDA version to 12.2, and CUDA 11 needs to be installed by yourself. I'll see if I can upgrade it when I have time.

**Comment by Cosplayeridol** [comment]
I am currently using this project for work and I pray that you can fix it as soon as possible. If you fix it, can you please reply to me in the comment area?

**Comment by lewangdev** [comment]
Ok.

**Comment by lewangdev** [comment]
I tried, but failed. I'll think of another way. Because it's the weekend and I have to be with my child, I don't have time.

**Comment by Cosplayeridol** [comment]
I understand very well, and I will wait patiently for you to fix the project.🥹

**Comment by lewangdev** [comment]
Fixed at 1dcfc4c94fa9e6c4026ea8f1a63509775b547438
